iina/Base.lproj/InspectorWindowController.xib

```diff
@@ -1124,7 +1124,7 @@
                         </tabViewItems>
                     </tabView>
                     <segmentedControl verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="0rt-Td-kr8">
-                        <rect key="frame" x="111" y="371" width="202" height="20"/>
+                        <rect key="frame" x="116" y="371" width="193" height="21"/>
```

Comment:
Did you changed this on purpose (small segment controls should have a height of 20)? If not, I'll discard this file when merging.